doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
+- nightly 
+- release
+ 
+You can manually add config for yum repo or download it from FIWARE public repository.
+
+Use this config for nightly packages:
+```
+[fiware-nightly]
+name=FIWARE nightly repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/nightly
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo)
+```
+curl -o /etc/yum.repos.d/fiware-nightly.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-nightly.repo
+
+```
+
+Use this config for release packages:
+```
+[fiware-release]
+name=FIWARE release repository
+baseurl=http://nexus.lab.fiware.org/repository/el/$releasever/$basearch/release
+enabled=1
+gpgcheck=0
+priority=1
+
+```
+or download it from [FIWARE public repository](http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-release.repo)
+```
+curl -o /etc/yum.repos.d/fiware-release.repo http://nexus.lab.fiware.org/repository/raw/public/repositories/el/7/x86_64/fiware-release.repo
+```
+
+Keep in mind, configs from FIWARE public repository have https connection and this can cause problems, to avoid them see [SSL configuration](#SSL)
```

Comment:
You menation https here but the URLs used in baseurl= above use http... Is that ok? Maybe I'm missing something?

Reply:
We have a problem, that the certificate may not be in root chain, so https connection from scratch will fail. So if you want to use https, you have to add certificate before using yum/curl/wget. All repos for yum/raw purposes are available for both type of the connections.

Reply:
If I'm understanding correctly, the the paragraph in the PR should be adjusted this way:

> Note the above `baseurl=` use `http`. However, if you want to change them to `https` you could have some issues. To avoid them see...

Does it make sense? Or maybe I'm understanding incorrectly?

Reply:
Will move everything to https

Reply:
fixed in ce41e406315343b6c61358ad41eda2854b4a1b63